### DISCUSSION TITLE
Tips for where is best to apply the curves

### DISCUSSION THREAD

**Discussion by Dan017** [Rem0o/FanControl.Releases#639] [Q&A]
Hi. I need an advice from you guys: Where and which curve is the best to apply to CPU Fan, SYSFAN HUB (in my case)? CPU Temp or Mix? Or how did u guys set it?
P.S. First time i try something like that ahah so i am also opened for other tips and tricks whatever the type of them.
![Capture](https://user-images.githubusercontent.com/104276294/164944125-8f911988-5e2f-413c-9e04-9ff42109e22f.PNG)
![Capture](https://user-images.githubusercontent.com/104276294/164944159-e8c49d01-53de-439c-8814-346019ae99bd.PNG)
![mix](https://user-images.githubusercontent.com/104276294/164944169-9b967034-5297-4d61-b1ec-62b21eb0f511.PNG)

**Comment by Fr0stX76** [comment]
it hasnt been updated for a bit, but still relevent for graph types

https://www.youtube.com/channel/UCClIDYAEncCD9d-Kmxumj3A/videos

**Reply by Dan017** [reply]
Thank you very much!

**Comment by xhul-dev** [comment]
Assuming those curves are relevant according to your hardware, the wisest option would be "CPU Temp" for "CPU Fan", and "Mix" for "SYSFAN HUB".
It guarentees the GPU will get fresh intake air according to its temp, and the hot air it creates gets exhausted properly.

**Comment by Rem0o** [comment]
Also, pair FAN1 Nvidia with its RPM sensor, it's the only 1 missing. Then, use the hamburger menu to tick the checkbox to hide the speed cards.

Will give you a cleaner look.

Finally, from the hamburger menu, set your colors/ theme how you like it.

**Reply by Dan017** [reply]
I left the speed cards there just for informative reasons😄. It didnt seem to work when i clicked to pair the sensor... Then after some tries it worked. Everything s fine. Thanks.
